README.rst

```diff
@@ -766,6 +770,19 @@ variable ``FUN``:
         Note that ``OPTS['description']`` must be defined and hold a non-empty
         string.
 
+One package is generated by package definition file. The package definition
+file used by default will be in ``PKG_FILES``. This is useful if one wants
```

Comment:
To be honest the first 2 sentences are not clear to me. And also there is a part above where it says all packages in `pkg/*` will be built:
```rst
The predefined ``pkg`` target will scan for ``*.pkg`` files in the ``$(PKG)``
directory (by default ``$T/pkg``) and then invoke ``mkpkg`` with them.
```

What about moving this part next to that and unify if, something like:
```rst
The predefined ``pkg`` target depends on the special variable ``$(pkg)``,
where you can add any extra target that must be built for ``pkg``.
By default every package file specified in `$(PKG_FILES)` will be
added to ``$(pkg)``, and by default `$(PKG_FILES)` holds all
``*.pkg`` files in the ``$(PKG)`` directory (by default ``$T/pkg``).
``mkpkg`` is invoked for every file specified in `$(PKG_FILES)`.
```

Then the example below can be maybe kept here.